src/remote/run_jobs.jl

```diff
@@ -0,0 +1,198 @@
+using Snowflake
+using JSON
+using HTTP
+
+commonpath="benchmarking/data"
+
+function serialize_circuit(circuit::QuantumCircuit,repetitions::Integer;indentation::Integer=4)
+  
+    circuit_description=Dict(
+        "circuit"=>Dict{String,Any}(
+                "operations"=>Vector{Dict{String,Any}}()
+            ),
+        "num_repititions"=>repetitions
+    )
+
+    for gate in get_gates(circuit)
+        push!(
+            circuit_description["circuit"]["operations"],
+            Dict{String,Any}(
+                "type"=> get_instruction_symbol(gate),
+                "qubits"=> get_connected_qubits(gate),
+                "parameters"=> get_parameters(gate)  
+            )
+        )
+    end
+
+    circuit_json=JSON.json(circuit_description,indentation)
+
+    write(
+        joinpath(commonpath,"test_circuit.json"), 
+        circuit_json
+    )
+
+    return circuit_json
+end
+
+
+struct Client 
+    host::String
+    user::String
+    access_token::String
+end
+
+get_host(client::Client)= client.host
+
+# submits circuit to host and returns circuitID
+function submit_circuit(client::Client,circuit_json::String;verbose=false) 
```

Comment:
Submit circuit should take a circuit and number of repetitions. The fact the client serialized the circuit to JSON is an implementation detail.